src/cpsr.rs

```diff
@@ -1,3 +1,5 @@
+#[cfg(test)] // TODO: remove cfg when this API will be used at least one in prod code.
+use crate::condition::ModeBits;
```

Comment:
If we need this only in test just put under `mod test`

Reply:
I understand that maybe I've written to many lines of code.
However as you can see `ModeBits` is used at line 157, but since the function isn't used at the moment I had to mark `use crate::condition::ModeBits;` with `#[cfg(test)]`

![Immagine 2022-09-13 235810](https://user-images.githubusercontent.com/99926684/190016993-c5e18f5d-895a-47a2-bb5b-4420c98a7a8c.png)

Reply:
Ok, if it isn't used you can mark this as `#[allow(dead_code)]`... But ok I understood your goal.

Reply:
Ok, perfect!